src/convertToObject.js

```diff
@@ -6,7 +6,25 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const resultObject = {};
+  let key = '';
+  let value = '';
+
+  const splitAllstring = sourceString.split(';');
+
+  for (const block of splitAllstring) {
+    const splitBlock = block.split(':');
+
+    key = splitBlock[0];
+    value = splitBlock[1];
```

Comment:
There is no need to declare 'key' and 'value' outside of the loop. These can be declared within the block scope where they are being used, which is more in line with modern JavaScript practices.